server/src/main/java/org/candlepin/model/ConsumerCurator.java

```diff
@@ -79,19 +79,17 @@ public ConsumerCurator() {
     @Transactional
     @Override
     public Consumer create(Consumer entity) {
```

Comment:
Almost. This method need not exist -- only the two-method version with flush. In AbstractHibernateCurator we know it calls flush(entity, bool), so we can just override that one method and cover both.